Task 3/script.js

```diff
@@ -0,0 +1,55 @@
+/* ------------------------------ TASK 3 -----------------------------------
+Parašykite JS kodą, kuris leis vartotojui paspaudus ant mygtuko "Show users"
+pamatyti vartotojus iš Github API (endpoint'as pateiktas žemiau).
+
+Paspaudus mygtuką "Show users":
+1. Informacija atvaizdavima <div id="output"></div> bloke
+1.1. Informacija, kuri pateikiama: "login" ir "avatar_url" reikšmės (kortelėje)
+2. Žinutė "Press "Show Users" button to see users" turi išnykti;
+
+Pastaba: Sukurta kortelė, kurioje yra pateikiama vartotojo informacija, turi 
+turėti bent minimalų stilių ir būti responsive;
+-------------------------------------------------------------------------- */
+
+const ENDPOINT = 'https://api.github.com/users';
+const SHOW_USERS_BUTTON = document.getElementById('btn');
+const MESSAGE_P_TAG = document.getElementById('message');
+const OUTPUT_TAG = document.getElementById('output');
+
+function fetchData() {
```

Comment:
this fetchData function basically does not do anything. If you would use async/await functionality it would have purpose, but now you could just move everything outside.